vnode_controller/vnode_controller.go

```
@@ -147,120 +142,63 @@
-		log.G(ctx).WithError(err).Error("unable to watch VNodes")
-		return err
-	}
-
 	go func() {
 		// wait for all tunnel to be ready
 		utils.CheckAndFinallyCall(context.Background(), func() bool {
-			for _, t := range brc.tunnels {
-				if !t.Ready() {
-					return false
-				}
+			if !vNodeController.tunnel.Ready() {
```

golangci-lint

S1008: should use 'return vNodeController.tunnel.Ready()' instead of 'if !vNodeController.tunnel.Ready() { return false }; return true'

(gosimple)